tftest.py

```diff
@@ -0,0 +1,14 @@
+import gensim,os,time 
```

Comment:
Doesn't belong in repo.

If meant to be included, the code has to properly formatted (see other CLI script in the `scripts` directory).